specification/marketplacecatalog/resource-manager/Products/readme.md

```diff
@@ -0,0 +1,39 @@
+# marketplacecatalog
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for marketplacecatalog.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the marketplacecatalog.
+
+```yaml
+openapi-type: arm
+tag: package-2022-09-01-preview
+```
+
+### Tag: package-2022-09-01-preview
+
+These settings apply only when `--tag=package-2022-09-01-preview` is specified on the command line.
```

Comment:
I didn't see any configuration with SDK generation, is this expected?

Reply:
Not sure I understand the question. This file relates to autorest